app/forms/location_form.rb

```diff
@@ -0,0 +1,25 @@
+class LocationForm < BaseForm
+  attr_accessor :location
+
+  def initialize(transient_registration)
+    super
+    self.location = @transient_registration.location
+  end
+
+  def submit(params)
+    # Assign the params for validation and pass them to the BaseForm method for updating
+    self.location = params[:location]
+    attributes = { location: location }
+
+    # Set the business type to overseas when required as we use this for microcopy
+    attributes[:business_type] = "overseas" if location == "overseas"
+
+    super(attributes, params[:reg_identifier])
+  end
+
+  validates :location, inclusion: { in: %w[england
```

Comment:
My one pondering is whether there is a call to centralise lists such as this (business type being the other example) but as yet can't point at a specific reason why. So more a question of whether you've had any similar thoughts.

Reply:
@Cruikshanks This had crossed my mind as well. I guess one benefit would be that when we have a form with radio buttons for each of these options, we could iterate over the existing list rather than adding them all manually. What do you think? Shouldn't take too long to implement, although I'd probably do it as a separate PR if we're doing business types too.